task_id_display/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Task ID Display
+===============
+
+This module extends the functionality of project and allow you to see the task ID from the task views.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (project, project_issue)
```

Comment:
project_issue is inherited from project.
I thinks we don't need install project here.